client/packages/admin/src/index.ts

```diff
@@ -231,6 +233,33 @@ function steps(inputChunks) {
   return chunks.flatMap(getOps);
 }
 
+type PresenceResult<Data> = {
+  [peerId: string]: { data: Data; 'peer-id': string; user: User | null };
+};
+
+class Rooms<Schema extends InstantSchemaDef<any, any, any>> {
+  config: FilledConfig;
+
+  constructor(config: FilledConfig) {
+    this.config = config;
+  }
+
+  async getPresence<RoomType extends keyof RoomsOf<Schema>>(
```

Comment:
Uncertain about the naming here. 

Some options: 

- 1. db.rooms.getPresence(...) 
- 2. db.getRoom(...)
- 3. db.rooms.getSessions(...) 

Thoughts @dwwoelfel @nezaj @tonsky ?